scripts/index.ts

```diff
@@ -22,7 +24,21 @@ function initializeConfigDefaults(config) {
     };
 }
 
-body.ready(() => {
+let started = false;
+
+export function startApp(
+    customUiComponents: IConfigurableUiComponents,
+) {
+    if (started === true) {
+        return;
+    }
+
+    started = true;
+
+    for (const key in customUiComponents) {
+        CC[key] = customUiComponents[key];
+    }
```

Comment:
It's a bit safer to include a check here for `if (customUiComponents.hasOwnProperty(key)` before the assignment, so you only loop and assign the instance keys and not the unknown keys from the prototype chain.